src/main/java/com/example/board/domain/member/controller/MemberController.java

```diff
@@ -0,0 +1,41 @@
+package com.example.board.domain.member.controller;
+
+import com.example.board.domain.member.dto.MemberRequest;
+import com.example.board.domain.member.dto.MemberResponse;
+import com.example.board.domain.member.service.MemberService;
+import java.net.URI;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/member")
+public class MemberController {
```

Comment:
```suggestion
@RestController
@RequestMapping("/members")
public class MemberController {
```

REST API에서는 자원을 복수로 표현합니다

https://meetup.toast.com/posts/92

Reply:
최근에 여러 블로그들 보면 다들 복수로 표현하는 것을 확인했습니다. RESTful API는 자원을 복수로 표현하는게 기준이군요!!